06-management/03-migration-and-backup.md

```diff
@@ -0,0 +1,110 @@
+---
+pageTitle: Migration and Backup
+keywords: grakn, migration, backup
+longTailKeywords: grakn migration
+Summary: Backing up and migrating data between versions of Grakn.
+toc: false
+---
+
+# Migration and Backup
+
+Some versions of Grakn include tools to backup and migrate data between versions of grakn that are not data-level compatible. Please note the chart below for which versions are data-level compatible:
+
+| Data-level Versions | Versions at Data-level | Latest Version With Migration Available |
```

Comment:
shouldn't this be "Earliest Version with Migration Available" ?

Reply:
Yes